samples/XCT.Sample.Android/Xamarin.CommunityToolkit.Sample.Android.csproj

```diff
@@ -22,6 +22,9 @@
     <AndroidHttpClientHandlerType>Xamarin.Android.Net.AndroidClientHandler</AndroidHttpClientHandlerType>
     <NuGetPackageImportStamp>
     </NuGetPackageImportStamp>
+    <Nullable>enable</Nullable>
+    <WarningsAsErrors>nullable</WarningsAsErrors>
+    <LangVersion>latest</LangVersion>
```

Comment:
It's better to move it to [Directory.Build.props](https://github.com/xamarin/XamarinCommunityToolkit/blob/main/Directory.Build.props) instead of adding it to every project file directly.

PS: `LangVersion` is already in there